modules/siddhi-core/src/test/java/org/wso2/siddhi/core/stream/output/sink/LogSinkTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c)  2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.siddhi.core.stream.output.sink;
+
+import org.apache.log4j.Logger;
+import org.junit.Test;
+import org.wso2.siddhi.core.SiddhiAppRuntime;
+import org.wso2.siddhi.core.SiddhiManager;
+import org.wso2.siddhi.core.stream.input.InputHandler;
+
+public class LogSinkTest {
+    private static final Logger logger = Logger.getLogger(LogSinkTest.class);
+
+    @Test
+    public void testLogSink_1() throws Exception {
+        logger.info("LogSink Test 1 - With all options");
```

Comment:
Instead of putting a log, rename the test method to be meaningful & more descriptive e.g. `testWithAllOptions`. Same goes for other test methods as well.

Reply:
done